### ISSUE TITLE
v16.11.0 broken

### ISSUE THREAD

**Issue by daker** [Kitware/vtk-js#1769]
For some reasons the cone example is broken after this release 

https://codepen.io/jourdain/pen/RQZWYa

```js
vtk.js:2 Uncaught ReferenceError: Constants is not defined
    at Module.2782 (vtk.js:2)
    at n (vtk.js:2)
    at vtk.js:2
    at vtk.js:2
    at vtk.js:2
    at vtk.js:2
```

**Comment by martinken** [comment]
I fixed it locally, probabel something like 
```
diff --git a/Sources/Rendering/WebGPU/BufferManager/index.js b/Sources/Rendering/WebGPU/BufferManager/index.js
index ed712b2689..0d139a715a 100644
--- a/Sources/Rendering/WebGPU/BufferManager/index.js
+++ b/Sources/Rendering/WebGPU/BufferManager/index.js
@@ -2,7 +2,9 @@ import macro from 'vtk.js/Sources/macro';
 import * as vtkMath from 'vtk.js/Sources/Common/Core/Math';
 import vtkWebGPUBuffer from 'vtk.js/Sources/Rendering/WebGPU/Buffer';
 import vtkProperty from 'vtk.js/Sources/Rendering/Core/Property';
-import { BufferUsage, PrimitiveTypes } from './Constants';
+import Constants from 'vtk.js/Sources/Rendering/WebGPU/BufferManager/Constants';
+
+const { BufferUsage, PrimitiveTypes } = Constants;
```

**Comment by jourdain** [comment]
Fixed in 16.11.1
